### PR TITLE
WorldsTest Fix

### PR DESCRIPTION
**Pull request by LOLDERP**
changed some stuff in WorldsTest

### CODE REVIEW

File: Plugin/WorldsTest/src/main/java/org/spongepowered/cookbook/plugin/WorldsTest.java [hunk 1]
```diff
@@ -88,26 +88,26 @@ public void onServerAboutToStart(GameAboutToStartServerEvent e) {
...
-                        }
-                        for (Player target : args.<Player>getAll("target")) {
-                            target.setLocation(new Location(optWorld.get(), optWorld.get().getProperties()
-                                    .getSpawnPosition()));
-                        }
-                        return CommandResult.success();
-                    }
-                })
-                .build()
+                        .description(Texts.of("Teleports a player to another world"))
```
Comment (gabizou) [inline Plugin/WorldsTest/src/main/java/org/spongepowered/cookbook/plugin/WorldsTest.java]: This is formatting noise.

File: Plugin/WorldsTest/src/main/java/org/spongepowered/cookbook/plugin/WorldsTest.java [hunk 2]
```diff
@@ -213,4 +213,4 @@ public CommandResult process(CommandSource source, String arguments) throws Comm
             return null;
         }
     }
-}
+}
```
Comment (gabizou) [inline Plugin/WorldsTest/src/main/java/org/spongepowered/cookbook/plugin/WorldsTest.java]: How do you delete a new line at the end of file intentionally?

File: Plugin/WorldsTest/build.gradle [hunk 3]
```diff
@@ -1,18 +1,11 @@
...
-        name 'Sponge maven repo'
-        url 'http://repo.spongepowered.org/maven'
+        name = 'sponge'
+        url = 'http://repo.spongepowered.org/maven'
     }
 }
 
 dependencies {
-    compile "org.spongepowered:spongeapi:2.1-SNAPSHOT"
+    compile 'org.spongepowered:spongeapi:2.1-SNAPSHOT'
```
Comment (gabizou) [inline Plugin/WorldsTest/build.gradle]: Why?

File: Plugin/WorldsTest/build.gradle [hunk 4]
```diff
@@ -1,18 +1,11 @@
-apply plugin: 'java'
```
Comment (Aaron1011) [inline Plugin/WorldsTest/build.gradle]: Why did you remove this?

File: Plugin/WorldsTest/build.gradle [hunk 5]
```diff
@@ -1,18 +1,11 @@
...
-
 repositories {
-    mavenLocal()
     mavenCentral()
     maven {
-        name 'Sonatype'
```
Comment (gabizou) [inline Plugin/WorldsTest/build.gradle]: Not sure why you're removing this.

**Comment by LOLDERP** [comment]
this was to be a branch not a PR i did a PR by accident
